### PR TITLE
Remove renv and other sundry items

### PR DESCRIPTION
**Pull request by bms63**

### CODE REVIEW

**Comment by bms63** [comment]
Probably should set up dev version of website and vbump stufff....ummm what else??

**Comment by rossfarrugia** [comment]
Thanks @bms63 - i'm a little out of date with all the latest admiral conventions here so i trust your updates and maybe @manciniedoardo could double check no others needed (same as for admiraltemplate PR: https://github.com/pharmaverse/admiraltemplate/pull/72). Then I can focus on getting some pieces organised ahead of the kick-off, like a project board and some early issues or discussions for people to start contributing to.

I did address the PR template update as you commented at https://github.com/pharmaverse/admiraltemplate/issues/75

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-- [ ] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
-- [ ] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
+- [ ] Update `NEWS.md` under the header `# admiralpeds (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
+- [ ] Build admiralpeds site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
 - [ ] Address or fix all lintr warnings and errors - `lintr::lint_package()`
 - [ ] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
 - [ ] Link the issue in the Development Section on the right hand side.
-- [ ] Address all merge conflicts and resolve appropriately 
+- [ ] Address all merge conflicts and resolve appropriately
```
Comment (manciniedoardo) [inline .github/pull_request_template.md:17]: Might want to add a note about bumping the package version as well (see my comment in [this PR](https://github.com/pharmaverse/admiraltemplate/pull/72#pullrequestreview-1872726088))

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Looks good, just  a note about versioning and also a thought: would it be good to already put the admiralpeds hex in the repo so that it's available when the pkgdown site is updated?

File: .github/pull_request_template.md (hunk 1 above)
Comment (bms63) [reply .github/pull_request_template.md:17]: yes we should add

Comment (bms63) [reply .github/pull_request_template.md:17]: i made into separate issue - not entirely sure where to put it

**Pull request merged**